app/MMB/src/components/Alert.js

```diff
@@ -33,17 +33,19 @@ const RenderAlert = (props: RenderAlertProps) => (
     <View style={styles.iconContainer}>{props.icon}</View>
     <View>
       <Text style={[styles.titleText, props.titleStyle]}>{props.title}</Text>
-      <View style={styles.body}>{props.children}</View>
+      {props.children && <View style={styles.body}>{props.children}</View>}
     </View>
   </View>
 );
 
+const ICON_SIZE = 17;
+
 export default function Alert(props: Props) {
   switch (props.type) {
     case 'danger':
       return (
         <RenderAlert
```

Comment:
Is it time to move `RenderAlert` somewhere else already or not yet? 😊

Reply:
`@kiwicom/mobile-alert`?

Reply:
No, new file! 🙈

Reply:
done